pipeline/backend/kubernetes/pod.go

```diff
@@ -31,8 +31,11 @@ import (
 )
 
 const (
-	StepLabel = "step"
-	podPrefix = "wp-"
+	StepLabel           = "step"
+	podPrefix           = "wp-"
+	PipelineNumberLabel = "pipeline_number"
```

Comment:
As we use [dashes](https://github.com/woodpecker-ci/woodpecker/blob/134fb7900c514d4498ce02767715bb16d0d46f36/pipeline/backend/kubernetes/utils_test.go#L26), I suggest to change to `pipeline-number` and `workflow-name`.